storybookjs/storybook
# Manager: Fix cannot read properties of undefined at SearchResults

Closes #21623

## What I did
I just got rid of the console.log by using nullish coalescing.

But I'm wondering if the `mouseOverHandler` is necessary at all, because the condition at L210 will never be met.
The `getData` function will return `undefined` for stories of type `component`, so`isComponent` will always be false (at `/modules/stories.ts`). Besides, I saw `isComponent` is tagged as @deprecated at `API_LeafEntry` and is typed as false (at `/modules/api-stories.ts`). 

I'm lacking context to make further changes though. I'd appreciate if you can point me in the right direction if more changes are needed.

## How to test
1. Run sandbox. 
2. At manager, focus "Find components" input, "RECENTLY OPENED" stories list is shown. 
3. If you hover any stories in that list you should not see errors in the console.

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
